terraform/base-infra/WIREGUARD_SETUP.md

```
@@ -206,21 +206,21 @@ After setting up WireGuard, you need to create **multiple peer configurations**
 
 2. **CLUSTER_WIREGUARD_WG0 Secret:**
  - Name: `CLUSTER_WIREGUARD_WG0`
- - Value: Contents of `peer1.conf` 
+ - Value: Contents of `peer2.conf` 
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)